### PR TITLE
YG-77 Remove unutilized 3rd party libraries

### PR DESCRIPTION
**Pull request by pepcus-vikash**
### Task

- Removed all the unutilized 3rd party libraries and run yarn install. 
- App is running same as before. 
### Jira reference 
https://pepcus.atlassian.net/browse/YG-77
### Screenshot
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,36 +3,24 @@
...
-		"@fortawesome/fontawesome-svg-core": "1.2.18",
-		"@fortawesome/free-regular-svg-icons": "5.8.2",
 		"@fortawesome/free-solid-svg-icons": "5.8.2",
-		"@fortawesome/react-fontawesome": "0.1.4",
 		"@pepcus/react-simple-data-grid": "2.9.0",
-		"css-vars-ponyfill": "^2.0.2",
 		"csvtojson": "^2.0.8",
 		"deepmerge": "^4.1.1",
 		"es6-template-regex": "^1.0.0",
-		"file-saver": "^1.3.8",
```
Comment (pepcus-mohit) [inline package.json:15]: Can you re-verify this feature on the files screen?
Make sure it's working correctly.

File: package.json [hunk 2]
```diff
@@ -3,36 +3,24 @@
...
 		"font-awesome": "^4.7.0",
 		"fuse.js": "^3.0.5",
 		"js2excel": "^1.0.1",
-		"jsbarcode": "^3.11.0",
```
Comment (pepcus-mohit) [inline package.json:19]: Can you verify and confirm this feature is working on Print now button on the MemberInfoGrid screen.

Comment (pepcus-vikash) [reply package.json:19]: Print now button is working and processing the card as before :- 
image

react-barcode module is being used in the project.

File: package.json (hunk 1 above)
Comment (pepcus-vikash) [reply package.json:15]: Verified the files page and all the functionality seems to be working fine. 
image

**Review by pepcus-mohit — APPROVED** [review]

**Pull request merged**